server/api/views.py

```diff
@@ -242,3 +243,11 @@ def get(self, request, *args, **kwargs):
             return HttpResponseForbidden()
         else:
             return Response({'message': 'Authorization was a success'})
+
+class LaunchSlackAuthorization(APIView):
+    def get(self, request, *args, **kwargs):
+        client_id = getenv('SLACK_CLIENT_ID')
+        scope='read'
+        oauth_url = 'https://slack.com/oauth/authorize?client_id={}&scope={}'
```

Comment:
`oauth_url =f 'https://slack.com/oauth/authorize?client_id={client_id}&scope={scope}'`  ?? python 3 format should be prefered here. 

Reference:
https://docs.python.org/3/reference/lexical_analysis.html#f-strings
https://realpython.com/python-f-strings/